alliance_auth/urls.py

```diff
@@ -203,4 +204,18 @@
                        url(r'^notifications/$', 'notifications.views.notification_list', name='auth_notification_list'),
                        url(r'^notifications/(\w+)/$', 'notifications.views.notification_view', name='auth_notification_view'),
                        url(r'^remove_notifications/(\w+)/$', 'notifications.views.remove_notification', name='auth_remove_notification'),
-)
+
+                       # FleetActivityTracking (FAT)
+                       url(r'^fat/$', 'fleetactivitytracking.views.fatlink_view', name='auth_fatlink_view'),
```

Comment:
Because of recent geo-political events, there is a strong wish from certain people to not be connected to anything regarding Goons/IMP...